packages/kbn-test/src/page_load_metrics/navigation.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Fs from 'fs';
+import Url from 'url';
+import _ from 'lodash';
+import puppeteer from 'puppeteer';
+import { resolve } from 'path';
+import { ToolingLog } from '@kbn/dev-utils';
+import { ResponseReceivedEvent, DataReceivedEvent } from './event';
+
+export interface NavigationOptions {
+  headless: boolean;
+  appConfig: { url: string; username: string; password: string };
+  screenshotsDir?: string;
```

Comment:
Can we please require this option and only define the default in the CLI?

Reply:
Done in fec3c4ef864193579f9daf92bb793a1deb2995c0